### DISCUSSION TITLE
can `crypto_secretbox_detached` and `crypto_secretbox_open_detached` param `c` and `m` overlap?

### DISCUSSION THREAD

**Discussion by meiyese** [jedisct1/libsodium#1384] [General]
From the document `crypto_secretbox_easy` and `crypto_secretbox_open_easy` function params `c` and `m` can overlap.

how about `crypto_secretbox_detached` and `crypto_secretbox_open_detached`, can params `c` and `m` overlap?

**Comment by jedisct1** [comment]
Yes.
